scripts/postgres_install_moodle.sh

```diff
@@ -625,14 +625,11 @@ EOF
     /usr/bin/unzip -q moodle.zip
     /bin/mv -v moodle-'$moodleVersion' /moodle/html/moodle
 
-    # Commented out as the plugin is not available
     # install Office 365 plugins
-    #if [ "$installOfficePlugins" = "True" ]; then
```

Comment:
ditto the comment above for the MySQL install.

As an aside we should look to remove as much duplication between the postgres and MySQL install scripts as possible. No need to block this PR on it, but something to bear in mind.

Reply:
Absolutely. I mentioned about this code duplication and wanted to do that properly, but that'll require some nontrivial refactoring of the shell scripts and modularization of that. I gave it some thoughts how to achieve that, but I realized that it's actually nontrivial. This is in line with our discussion with Anand about how applications are deployed through ARM. The current CustomScript extension-based one is quite crude and clunky, and yet there doesn't seem to be a good alternative yet.

Reply:
on ACS we used [Cloud Init](https://docs.microsoft.com/en-us/azure/virtual-machines/linux/using-cloud-init) to great effect . I think it would serve our purpose here too. But again, we should make this separate from this PR which is valuable without that bigger change.